### PR TITLE
Chore/update appraisals

### LINKED ISSUE #918: Add more CIs (Rails 5.2, Webpacker v4 pre)
- [x] Against Rails 5.2 and Webpacker v3
- [x] Against Webpacker v4 pre (allow_failures)

### PR DESCRIPTION
**Pull request by BookOfGreg**
### Summary

Adds Rails 5.2 + Webpacker 3, 4.pre versions to appraisal, with Webpacker 4.pre set as allow fail.
Closes #918

### CODE REVIEW

File: Appraisals [hunk 1]
```diff
@@ -4,6 +4,18 @@ appraise 'rails-5.2-sprockets_4' do
   gem 'turbolinks', '~> 5'
 end
 
+appraise 'rails-5.2_no_sprockets_webpacker_3' do
+  gem 'rails', '~> 5.2.0'
+  gem 'webpacker', '4.0.0.pre.pre.2'
```
Comment (ttanimichi) [inline Appraisals]: Did you mean webpacker3?

**Comment by BookOfGreg** [comment]
@ttanimichi Appears to be some failing test unrelated to this appraisal change. Must say in general I'm finding Travis a little flakey at times but this one does seem like a real failure on Webpacker 3.

**Comment by BookOfGreg** [comment]
This grid in Travis is roughly what I'm aiming for:

| Gemfiles\Rubies                    | 2.6.2 | 2.5.5 | 2.3.8 | jruby-9.1.9.0 |
|------------------------------------|-------|-------|-------|---------------|
| rails_4.2_sprockets_4              | x     |       |       |               |
| rails_5_no_sprockets_webpacker_1_1 | x     |       |       |               |
| rails_5_no_sprockets_webpacker_1_x | x     |       |       |               |
| rails_5_no_sprockets_webpacker_2   | x     |       |       |               |
| rails_5_no_sprockets_webpacker_3   | x     |       |       |               |
| rails_5.2_no_sprockets_webpacker_3 | x     |       |       |               |
| rails_5.2_no_sprockets_webpacker_4 | x     | x     | x     | Allow Fail    |
| rails_4.2_sprockets_2              | x     |       |       |               |
| rails_4.2_sprockets_3              | x     |       |       |               |
| rails_5_no_sprockets               | x     |       |       |               |
| rails_5.1_sprockets_4              | x     |       |       |               |
| rails_5.2_sprockets_4              | x     | x     | x     | Allow Fail    |

**Comment by BookOfGreg** [comment]
Reverted as Ruby 2.6.2 can't seem to `gem install webdrivers`
```
Parsing documentation for childprocess-0.9.0
Before reporting this, could you check that the file you're documenting
has proper syntax:

  /Users/me/.rbenv/versions/2.6.2/bin/ruby -c lib/childprocess/tools/generator.rb

RDoc is not a full Ruby parser and will fail when fed invalid ruby programs.

The internal error was:

	(NoMethodError) undefined method `[]' for nil:NilClass

ERROR:  While executing gem ... (NoMethodError)
    undefined method `[]' for nil:NilClass
```

**Comment by 0xdevalias** [comment]
With this merged now, does that mean webpacker 4.x is actually supported? If so.. should the README get updated to say so?

**Comment by BookOfGreg** [comment]
@0xdevalias thanks for asking for clarity on this.

Webpacker 4.x is supported in that it didn't crash the tests. I haven't personally tried it yet as my time is limited.
If you vouch it currently works (or can report bugs specifically for Webpacker 4.x) then yes we can call it supported.

**Pull request merged**